modules/mono/glue/GodotSharp/GodotSharp/Core/Plane.cs

```diff
@@ -289,6 +281,17 @@ public Plane(real_t a, real_t b, real_t c, real_t d)
             D = d;
         }
 
+        /// <summary>
+        /// Constructs a <see cref="Plane"/> from a <paramref name="normal"/> vector.
+        /// The plane will intersect the origin.
+        /// </summary>
+        /// <param name="normal">The normal of the plane, must be normalized.</param>
```

Comment:
```suggestion
        /// <param name="normal">The normal of the plane, must be a unit vector.</param>
```
Normalized vector is the same as unit vector so the meaning doesn't change. However, I think it's a better wording because of the `Normalized` method which can return a not normalized vector (for a zero vector it returns a zero vector). Meaning `someVector.Normalized()` is not guaranteed to satify this requirement. But seeing "must be normalized" users could intuitively assume otherwise. With suggested wording maybe it will be a little less probable.

Reply:
It's also weird that Core documentation doesn't have this line, I guess it should be added.